### PR TITLE
[FSSDK-8940] Correct return type hints

### PR DESCRIPTION
**Pull request by rafinutshaw-optimizely**
## Summary
- Check if testcases pass after fixing the hints as suggested in the PRs of the issue

## Test plan
- No changes here

## Issues
- https://jira.sso.episerver.net/browse/FSSDK-8940

### CODE REVIEW

File: src/Optimizely/OptimizelyUserContext.php
Comment (mikechu-optimizely) [inline src/Optimizely/OptimizelyUserContext.php:1]: @rafinutshaw-optimizely We'll need to increment the Copyright notice too for each file.

`Copyright 2021-2023, Optimizely`

File: src/Optimizely/UserProfile/UserProfileServiceInterface.php
Comment (mikechu-optimizely) [inline src/Optimizely/UserProfile/UserProfileServiceInterface.php:1]: I think this one should be: `Copyright 2017, 2023, Optimizely Inc, and Contributors`

Is that cool?

**Review by mikechu-optimizely — CHANGES_REQUESTED** [review]
Please review about copyright notices.

**Comment by mikechu-optimizely** [comment]
This PR once merged and released should resolve: #250, #254, & #259

File: src/Optimizely/UserProfile/UserProfileServiceInterface.php
Comment (rafinutshaw-optimizely) [reply src/Optimizely/UserProfile/UserProfileServiceInterface.php:1]: yep, missed the copyright changes. thanks for pointing them out

**Pull request merged**